analyzers/src/SonarAnalyzer.Common/Helpers/KnownType.cs

```diff
@@ -101,6 +101,7 @@ internal sealed partial class KnownType
         internal static readonly KnownType Microsoft_Extensions_Logging_LoggerExtensions = new("Microsoft.Extensions.Logging.LoggerExtensions");
         internal static readonly KnownType Microsoft_Extensions_Primitives_StringValues = new("Microsoft.Extensions.Primitives.StringValues");
         internal static readonly KnownType Microsoft_Net_Http_Headers_HeaderNames = new("Microsoft.Net.Http.Headers.HeaderNames");
+        internal static readonly KnownType Microsoft_VisualBasic_Information = new("Microsoft.VisualBasic.Information");
```

Comment:
What about other helper methods like, `object.ReferenceEquals(o, null)` or `object.Equals(o, null)`?

Reply:
FP/FN is adequate punishments for using those 😆